### PR TITLE
{lib}[foss/2022a] DeepLabCut v2.3.6 with CUDA 11.7.0

### PR DESCRIPTION
**Pull request by jhein32**

### CODE REVIEW

File: easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.3.6-foss-2022a-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+homepage = 'http://www.mousemotorlab.org/deeplabcut'
+description = "Markerless tracking of user-defined features with deep learning"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('Python', '3.10.4'),
+    ('TensorFlow', '2.9.1',  versionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.3.6-foss-2022a-CUDA-11.7.0.eb:21]: I'm a bit confused as to why the easyconfigs test suite isn't tripping over this, but TensorFlow 2.11.0 is the usual dependency in the `2022a` generation of easyconfigs, any reason you're not using that?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3902.accelgor.os - Linux RHEL 8.8, x86_64, AMD EPYC 7413 24-Core Processor (zen3), 1 x NVIDIA NVIDIA A100-SXM4-80GB, 535.104.12, Python 3.6.8
See https://gist.github.com/boegel/80c76a48026731a5f5f9c261057a74c2 for a full test report.

**Comment by boegel** [comment]
@jhein32 Some dependencies seem to be missing, like `cuDNN` and `OpenCV`, which are there for the existing `DeepLabCut` easyconfig, any reason you excluded those here?

**Comment by jhein32** [comment]
> @jhein32 Some dependencies seem to be missing, like `cuDNN` and `OpenCV`, which are there for the existing `DeepLabCut` easyconfig, any reason you excluded those here?

These are dependencies of packages which I declare as dependencies.  So trying to keep the config minimal I do not think they should be there.

File: easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.3.6-foss-2022a-CUDA-11.7.0.eb (hunk 1 above)
Comment (jhein32) [reply easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.3.6-foss-2022a-CUDA-11.7.0.eb]: I revised it for TF 2.11

**Comment by jhein32** [comment]
Anyone could have a look for outstanding issues?

**Comment by verdurin** [comment]
I've set a build going - still waiting for TF...

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/c562dbdb69c6ba97b2dd165678c5d123 for a full test report.

**Comment by jhein32** [comment]
Any love for this?

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 9 out of 9 (1 easyconfigs in total)
b-cn1603.hpc2n.umu.se - Linux Ubuntu 22.04, x86_64, AMD EPYC 7313 16-Core Processor, 1 x NVIDIA NVIDIA A100 80GB PCIe, 525.147.05, Python 3.10.12
See https://gist.github.com/akesandgren/940ac2ebaf464e2ceb7da93c5e437543 for a full test report.

**Review by akesandgren — APPROVED** [review]
LGTM

**Pull request merged**